docs/api-reference/io-columns-multiclass-classification.md

```diff
@@ -6,4 +6,4 @@ This trainer outputs the following columns:
 | Output Column Name | Column Type | Description|
 | -- | -- | -- |
 | `Score` | vector of <xref:System.Single> | The scores of all classes. Higher value means higher probability to fall into the associated class. If the i-th element has the largest value, the predicted label index would be i. Note that i is zero-based index. |
-| `PredictedLabel` | <xref:System.UInt32> | The predicted label's index. If it's value is i, the actual label would be the i-th category in the key-valued input label type. Note that i is zero-based index. |
+| `PredictedLabel` | [key](xref:Microsoft.ML.Data.KeyDataViewType) type | The predicted label's index. If it's value is i, the actual label would be the i-th category in the key-valued input label type. Note that i is zero-based index. |
```

Comment:
>key [](start = 22, length = 3)

Can we capitalize `[Key]` ?

Reply:
we can but then we need to make this change everywhere. Right now everywhere it is "key" and it is consistent. What do you suggest?

Reply:
it's --> its